Adding a note about setting the domain name.

Sfshaza opened the pull request:
Fixes https://github.com/flutter/flutter/issues/16670

Please review, @mit-mit 

This raises the question, how do you specify org? It's easy in Android Studio and the other JetBrains products. For command line, use `flutter create --org`. VSCode is a bit trickier. I spoke to Danny, and he added notes to this bug: https://github.com/Dart-Code/Dart-Code/issues/618. Bottom line, you can't easily do it in VSCode right now.

Later, I'd like to create a deployment page and include a section about setting the domain name. Then this note could link to that. (I don't want to further inflate the note on this page with the full info.)

mit-mit commented:
This seems a bit detailed for our getting started flow. How about moving this into the IDE documentation?
https://flutter.io/using-ide/#creating-projects
https://flutter.io/using-ide-vscode/#creating-projects

Sfshaza commented on get-started/test-drive.md:
```diff
@@ -163,6 +170,18 @@ You should see the updated string in the running app almost immediately.
+**Setting the company domain**<br>
+When creating a new app, some Flutter IDE plugins ask for an
+organization name in reverse domain order,
+something like `com.yourcompany`. Along with the name of the app,
```
Thanks for catching that!

DanTup commented:
I added some notes about setting org in Dart Code here:

https://github.com/Dart-Code/Dart-Code/issues/618#issuecomment-389772083

On reflection, I'm not sure putting an extra step for the domain in the `New Project` command would be best (for new users, it just gives them a non-trivial question to answer if it's a throwaway app) so for the next release they'll just be settings that the user can configure if they wish (org and languages) and they'll be used during the creation process silently.

Of course, we can refine in future; but I figured it's less controversial to add a new step later than add one now and then take it away.

Sfshaza commented:
Update: I moved the note to the Android Studio docs. We don't currently have a good solution for VSCode, but it's in the works.  For the command line, they can use the "flutter create --org" option.

mit-mit commented:
Content looks good, but please move it into the text itself and don't use an include when there is no need for it

Sfshaza commented:
I had added the text as an include, because I thought I'd also be adding it to the VSCode and command line pages. But the VSCode IDE doesn't yet have a solution for this (though it's in the works - see the issue Dan referenced above), and the command line doesn't have it's own page, other than the page in get started and this was deemed too detailed for that page.  So, I've removed it as an include. We can revisit later when we need this text elsewhere.
